dashboard/src/pages/packages/index.vue

```diff
@@ -85,13 +85,47 @@ const toggleLegend = () => {
                 >{{ pkg.name }}</router-link
               >
             </td>
-            <td><UUID :id="pkg.aipId" /></td>
+            <td>
+              <UUID :id="pkg.aipId" />
+            </td>
             <td>{{ $filters.formatDateTime(pkg.startedAt) }}</td>
-            <td><UUID :id="pkg.locationId" /></td>
-            <td><StatusBadge :status="pkg.status" /></td>
+            <td>
+              <UUID :id="pkg.locationId" />
+            </td>
+            <td>
+              <StatusBadge :status="pkg.status" />
+            </td>
```

Comment:
Not sure why this formatting changes, not a big deal of course.

Reply:
I figured out that I was using the default "Vue.volar" formatter instead of the VSCode "prettier" extension as configured in `.vscode/settings.json`.  After switching to prettier it reformatted.